TheMovieDB/TheMovieDBTests/TestsNetworkProvider.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  NetworkProviderTests.swift
+//  TheMovieDBTests
+//
+//  Created by Mariana Andrea Orman Berch on 7/10/21.
+//
+
+import XCTest
+@testable import TheMovieDB
+
+class TestsNetworkProvider: XCTestCase {
+    
+    var movies: [Movie] = []
+    
+    var imageView: UIImageView = UIImageView()
+    
+    let networkProvider = NetworkProvider()
+    
+    func testGetMovies() throws {
+        
+        networkProvider.getMovies(completion: { [weak self] result in
+            
+            self?.movies = result
+        })
+        
+        XCTAssertNotNil(movies)
+    }
```

Comment:
You need something else here so it can work correctly, the array will never be nil because you created it empty